test/gurka/test_node_access.cc

```diff
@@ -0,0 +1,73 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+
+TEST(Standalone, NodeAccess) {
+  const std::string ascii_map = R"(
+      A----B----C----D------------E
+           |         |            |
+           F         G            H
+           |         |            |
+           I----J----K------------L
+  )";
+
+  const gurka::ways ways = {
+      {"AB", {{"highway", "residential"}}}, {"BC", {{"highway", "residential"}}},
+      {"CD", {{"highway", "residential"}}}, {"DE", {{"highway", "residential"}}},
+      {"BF", {{"highway", "residential"}}}, {"DG", {{"highway", "residential"}}},
+      {"EH", {{"highway", "residential"}}}, {"FI", {{"highway", "residential"}}},
+      {"GK", {{"highway", "residential"}}}, {"HL", {{"highway", "residential"}}},
+      {"IJ", {{"highway", "residential"}}}, {"JK", {{"highway", "residential"}}},
+      {"KL", {{"highway", "residential"}}},
+  };
+
+  const gurka::nodes nodes = {{"F", {{"motor_vehicle", "no"}}}, {"G", {{"motorcar", "no"}}}};
+
+  const auto layout = gurka::detail::map_to_coordinates(ascii_map, 100);
+  auto map = gurka::buildtiles(layout, ways, nodes, {}, "test/data/gurka_node_access");
+  auto result = gurka::route(map, "A", "I", "auto");
```

Comment:
Suggestion: Instead of repeating the test for different costing models (where the expected result is the same), you can use [parametrized fixtures](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md) (see **Value-Parameterized Tests**) to pass-in the costing string as an arg to your test and then use that when making the route request.

Reply:
@mandeepsandhu I just simplified the tests since they are so simple.  Next time I will try out parameterized fixtures.  Ty for the suggestion.